enterprise/replicasync/replicasync.go

```diff
@@ -420,11 +420,13 @@ func (m *Manager) Close() error {
 			Time:  dbtime.Now(),
 			Valid: true,
 		},
-		RelayAddress: m.self.RelayAddress,
-		RegionID:     m.self.RegionID,
-		Hostname:     m.self.Hostname,
-		Version:      m.self.Version,
-		Error:        m.self.Error,
+		RelayAddress:    m.self.RelayAddress,
+		RegionID:        m.self.RegionID,
+		Hostname:        m.self.Hostname,
+		Version:         m.self.Version,
+		Error:           m.self.Error,
+		DatabaseLatency: m.self.DatabaseLatency,
+		Primary:         m.self.Primary,
```

Comment:
Was this a bug, now fixed?

Reply:
Now that I stare at this a little longer, this change itself may be a bug.
I don't see how it makes sense to have a non-zero database latency for a closed replica, or a closed replica be anything other than non-primary.